README.md

```diff
@@ -97,6 +97,17 @@ Port number the server runs on.
 
 Defaults to `8080`.
 
+###### -S, --secure
+Specify to serve over HTTPS.
+
+Disabled by default.
+
+###### -C, --certificatePath
```

Comment:
Ah, dang! Almost there! Just forgot to update the README after you changed the options back. Good to go after this 👍